### DISCUSSION TITLE
sv.plot_image using inches?!?

### DISCUSSION THREAD

**Discussion by AlainPilon** [roboflow/supervision#1423] [General]
I just realized `plot_image` `size` argument uses inches, with a default or 12x12. 

Other than making it easy to pass it to the underlying `math.pyplot.figure`, is there a specific reason to use inches? This is highly impractical for anyone working in a Jupyter notebook as the image resizes based on the window size. 

My suggestion would be to use a pixel based size with a default value using the original image size.

If for some reason we need to keep the size in inches (!), at least update the doc to replace: 

```
size (Tuple[int, int]): The size of the plot.
```

with 

```
size (Tuple[int, int]): The size of the plot in inches.
```

**Comment by LinasKo** [comment]
It indeed is a thin wrapper around `pyplot`, and that's the only reason inches are used. We don't want to drift from it quite yet.

I'll push a fix to clarify the docs, and add this to the list of smaller issues to look at in the future.

**Comment by AlainPilon** [comment]
Thanks. 

I had my working window rendering the picture ALMOST the same size as the original and it took me way too much time figuring out why my annotations didnt line up.

**Reply by SkalskiP** [reply]
You can always convert the image to Pillow and display it in collab. Just use `cv2_to_pillow`. I believe Pillow displays images in original resolution.
